app/models/annotation/AnnotationService.scala

```diff
@@ -180,15 +180,16 @@ object AnnotationService
     taskFox: Fox[Task],
     userId: BSONObjectID,
     tracingReferenceBox: Box[TracingReference],
-    dataSetName: String
+    dataSetName: String,
+    description: Option[String]
     )(implicit ctx: DBAccessContext) = {
 
     for {
       task <- taskFox
       taskType <- task.taskType
       tracingReference <- tracingReferenceBox.toFox
       _ <- Annotation(userId, tracingReference, dataSetName, task._team, taskType.settings,
```

Comment:
I would prefer to have one parameter per line if we have to break it. With this version I tend to oversee `_task` and `description`